### ISSUE TITLE
Notebook for measures of Central Tendency 

### ISSUE THREAD

**Issue by aabhapingle** [PetalsOnWind/Statistics-and-Econometrics-for-Data-Science#163]
Hi! I would like to create a new notebook for demonstrating the concept and use of mean, median, and mode. Kindly assign it to me. Thank you.

**Comment by PetalsOnWind** [comment]
@aabhapingle I expect a bit more of notebooks. Can you add what more things you can add. Maybe something about moments, mgf, pdf, pmf, etc?

**Comment by Tejal18** [comment]
@PetalsOnWind I can add the things that you have asked for that is the visualization of the central tendencies.
I want to contribute to this issue. I will create a notebook which will explain the central tendency concept by visualizing it.
Please do assign me this issue. Thanks.

**Comment by aabhapingle** [comment]
> @aabhapingle I expect a bit more of notebooks. Can you add what more things you can add. Maybe something about moments, mgf, pdf, pmf, etc?

@PetalsOnWind Yes, sure! I can add explanations for PMF, PDF, CDF and MGF and create visualizations for the same. I could also explain the applications of the same.
